Mathlib/Geometry/RingedSpace/Stalks.lean

```diff
@@ -225,7 +225,7 @@ def stalkIso {X Y : PresheafedSpace.{_, _, v} C} (α : X ≅ Y) (x : X) :
 set_option linter.uppercaseLean3 false in
 #align algebraic_geometry.PresheafedSpace.stalk_map.stalk_iso AlgebraicGeometry.PresheafedSpace.stalkMap.stalkIso
 
-@[simp, reassoc, elementwise]
```

Comment:
Why was this de-`simp`ed? Does the same comment apply as elsewhere about `simpNF` getting confused?

Reply:
Is a lemma with proof parameters that do not appear directly on the LHS (but a proof that uses it) appropriate as a `simp` lemma? If it should be, then I'll add a note.

Reply:
Yes, I think this is still fine as a simp lemma, since `simp` creates `x ⤳ y` as a side-goal. Did the linter complain?

Reply:
Yes, `simpNF` linter complains. I think this is because `x ⤳ y` doesn't appear directly on the LHS.

Reply:
Is `x ⤳ y` a proof as you said it was, or is it data?

Reply:
It's a proof https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Inseparable.html#Specializes